Sources/MapboxCoreNavigation/EHorizon/OpenLRStandard.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import MapboxNavigationNative
+
+/** Standard of OpenLR */
+public enum OpenLRStandard {
+
+    /**
+     [TomTom OpenLR](http://www.openlr.org/).
+
+     Supported references: line location, point along line, polygon.
+     */
+    case tomTom
```

Comment:
Consider a vendor-neutral case name: TomTom is one member of the OpenLR Association that defines this standard.

Reply:
Agreed to leave as is.